### PR TITLE
Containers have different margins

### PR DESCRIPTION
**Pull request by guidomodarelli**
### Description
Feature containers have different margins between them and the top containers. We want a design on which the containers separations are homogeneous
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/7027

### Evidence
image

### Test
Navigate to the overview page and ensure to check the margins.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Machi3mfl — DISMISSED** [review]
CR: ✅ 
Test: ✅ 

Screenshot 2024-09-30 at 2 10 11 PM

Screenshot 2024-09-30 at 2 09 48 PM

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
TEST: ✅ 

Captura de pantalla 2024-10-01 a la(s) 6 10 22 p  m

Captura de pantalla 2024-10-01 a la(s) 6 06 04 p  m

File: plugins/main/public/controllers/overview/components/stats.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 .vulnerabilites-summary-card {
   padding-top: 2vh;
 }
+
+.wz-welcome-stats {
```
Comment (asteriscos) [inline plugins/main/public/controllers/overview/components/stats.scss:5]: This class is not being used. Please delete it.

**Review by asteriscos — CHANGES_REQUESTED** [review]
CR: :red_circle: 
Test: :green_circle: 

All the cards have a total distance of 24 pixels with the adjacent boxes. :heavy_check_mark: 

Screenshot from 2024-10-08 12-55-31

**Review by asteriscos — APPROVED** [review]

**Review by Machi3mfl — APPROVED** [review]

**Review by Machi3mfl — APPROVED** [review]

File: plugins/main/public/components/common/welcome/welcome.scss [hunk 2]
```diff
@@ -1,14 +1,3 @@
-.wz-welcome-page .euiCard .euiTitle,
-.wz-module-body .euiCard .euiTitle {
-  font-size: 16px;
-  font-weight: 400;
-}
-
-.wz-welcome-page .euiCard .euiText,
-.wz-module-body .euiCard .euiText {
-  font-family: sans-serif;
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/welcome.scss:9]: **question**: Is unsetting the font-family styles intentional?

File: plugins/main/public/components/common/welcome/welcome.scss [hunk 3]
```diff
@@ -1,14 +1,3 @@
-.wz-welcome-page .euiCard .euiTitle,
-.wz-module-body .euiCard .euiTitle {
-  font-size: 16px;
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/welcome.scss:3]: **question**: Is unsetting the font-size styles intentional? I do not see any reference in the issue or pull request.

File: plugins/main/public/components/common/welcome/welcome.scss [hunk 4]
```diff
@@ -44,7 +33,6 @@ span.statWithLink:hover {
   text-decoration: underline;
 }
 
-.wz-welcome-page .flex-redirect-app-links {
-  display: flex;
-  flex-grow: 1;
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/welcome.scss:49]: **question**: Is unsetting the display and flex-grow styles intentional? I do not see any reference in the issue or pull request.

File: plugins/main/public/components/common/welcome/welcome.scss (hunk 3 above)
Comment (guidomodarelli) [reply plugins/main/public/components/common/welcome/welcome.scss:3]: The classes `.wz-welcome-page` and `.wz-module-body` are no longer in use.

File: plugins/main/public/components/common/welcome/welcome.scss (hunk 2 above)
Comment (guidomodarelli) [reply plugins/main/public/components/common/welcome/welcome.scss:9]: The classes `.wz-welcome-page` and `.wz-module-body` are no longer in use.

File: plugins/main/public/components/common/welcome/welcome.scss (hunk 4 above)
Comment (guidomodarelli) [reply plugins/main/public/components/common/welcome/welcome.scss:49]: The classes `.wz-welcome-page` and `.wz-module-body` are no longer in use.

**Pull request merged**